### ISSUE TITLE
Webhook issues

### ISSUE THREAD

**Issue by Kovaelin** [Fiaxhs/discord-irc#1]
I can't seem to get the webhook to work. Do I have to change any of the other files? Does it work if I use the Discord channel ID rather than the channel name?

My config file looks like this:
> [
> {
> "nickname": "ROBOBOT",
> "server": "IRC.URL.DNS",
> "discordToken": "TOkeN",
> "commandCharacters": ["!"],
> "ircStatusNotices": true,
> "ircOptions": {
> "floodProtection": false,
> "encoding": "utf-8"
> },
> "channelMapping": {
> "DISCORDIDNUMBERSA": "#IRC"
> },
> "webhooks": {
> "DISCORDIDNUMBERSA": "https://discordapp.com/api/webhooks/COPYPASTEA/COPYPASTEB"
> }
> },
> 
> {
> "nickname": "ROBOBOT2",
> "server": "IRC.URL.DNS",
> "discordToken": "TOkeN",
> "channelMapping": {
> "DISCORDIDNUMBERSB": "#IRC2"
> },
> "commandCharacters": ["!"]
> }
> ]

I'm also seeing this error:
> error: Received error event from Discord Error: read ECONNRESET
>     at exports._errnoException (util.js:1020:11)
>     at TLSWrap.onread (net.js:568:26)

**Comment by Fiaxhs** [comment]
> Do I have to change any of the other files?

Nope

> Does it work if I use the Discord channel ID rather than the channel name?

No. You must use the channel name, don't forget the `#`

>  I'm also seeing this error:

This is another issue and seems unrelated to this new feature.

**Comment by Kovaelin** [comment]
> This is another issue and seems unrelated to this new feature.

I originally thought the error might be related, because it doesn't show up when I ran @ekmartin's code without attempting to use the webhook, so I thought I'd mention it.


> This is another issue and seems unrelated to this new feature.

I made the change and also used the channel name for the lines outside the webhook to maintain consistency, but it still doesn't work. I also tested removing the webhook part from the .config file, and I'm still seeing the same error.
